1_python/1_getting-started/project-setup.md

```diff
@@ -25,4 +25,42 @@ Python project management tools offer similar dependency addition commands).
       language: bash
       code: |
         pdm add lmstudio
+    uv:
+      language: bash
+      code: |
+        uv add lmstudio
+```
```

Comment:
The fact I was editing this file reminded me that I wanted to explicitly add `uv` to the examples here, even though it's identical to the `pdm` invocation.